animations/spells/1st/chilling-spray.json

```diff
@@ -5,8 +5,6 @@
 			"preset": "template",
 			"file": "jb2a.cone_of_cold.blue",
 			"contents": [
-				{
```

Comment:
This adds another animation "as is". The intent here is to fill the entire 90° template with the animation which is only 30/45°

Reply:
Got it. Removing the nonempty-object check on the base schema and only applying it to the top-level object then.